### ISSUE TITLE
Complaints about binding to attribute values as string, where no binding is used

### ISSUE THREAD

**Issue by zoechi** [dart-lang/angular_analyzer_plugin#280]
![image](https://cloud.githubusercontent.com/assets/405837/25168860/5d3a2960-24e5-11e7-97e2-d65f438da66d.png)

I also got this for

    tabIndex="0"

**Comment by mk13** [comment]
We're looking into the `disabled` raising this, but I believe this is correct for `tabIndex`. I'm assuming `tabIndex` is bound to an int? If so, `tabIndex="0"` assigns a string value of "0" to it if the brackets are omitted and the error is being correctly raised.

**Comment by zoechi** [comment]
`tabIndex="0"` is valid without Angular and I think it should be valid with Angular.
IMHO it should only complain when `[]` or `{{}}` is used or when there is an input with that name.

**Comment by mk13** [comment]
My mistake, I forgot that tabIndex is a global attribute. This is a known issue and we're looking to resolve this issue along with all the other global attributes.

**Comment by MichaelRFairhurst** [comment]
Looks like I must have got the assignability test backwards or something.

I'm seeing this issue:

```
  @Input()
  dynamic stringVal;
  ...
<div stringVal="foo">...
```

where it also complains. This affects a lot of the material components library.

That said, tabindex is special, I'm not sure how that is working, or why it should.

**Comment by zoechi** [comment]
It seems if I use 
```html
tabindex="0"
```
instead of 

```html 
tabIndex="0"
```
it doesn't complain anymore.
I didn't expect casing to be relevant here. Not sure what the correct behavior would be.

**Comment by zoechi** [comment]
A bit weird is that if I change `I` to `i` the squiggled underline goes away, if I then change it back to `I` all squiggled underlines in this file go away and I have to restart DartAnalyzer to get the others back. Perhaps an unrelated issue though.

**Comment by MichaelRFairhurst** [comment]
That sounds like a crash.

I also know that casing does matter, because when we see an unknown property, we can't report a type error (for instance, it might be used for css only). Therefore changing the case probably just causes the input to fail to match up and therefore no type error is revealed

I might just disable this for global attributes entirely, but I'm still wary about things like

```
  <table caption="foo"
```

where `TableElement.caption` is an assignable field of type `HtmlCaptionElement`, not `String`.

That said, I have no idea how angular correctly maps the `int`s into `string`s, and what it means for us, and _that_ is pretty common!

**Comment by zoechi** [comment]
But if there is no binding involved and no input with a matching name (like with the `tabindex` example,, Angular doesn't do anything. It's just added to the DOM and processed by the browser. Or did I misunderstand your statement?

**Comment by MichaelRFairhurst** [comment]
Fixed in PR #306 

Thanks for filing, yet again, @zoechi  :)
